components/SiteNavbar.vue

```diff
@@ -0,0 +1,53 @@
+<template>
+  <nav class="flex flex-wrap flex-row-reverse justify-between border-b border-gray-300 px-5 py-3 lg:flex-row">
+    <div class="flex lg:hidden" @click="mobileView = !mobileView">
+      <button class="flex items-center border border-gray-300 rounded px-4 py-1 text-gray-600 hover:text-black hover:border-black hover:bg-gray-100">
+        <svg class="fill-current h-3 w-3" viewBox="0 0 20 20" xmlns="http://www.w3.org/2000/svg">
+          <title>Menu</title>
+          <path d="M0 3h20v2H0V3zm0 6h20v2H0V9zm0 6h20v2H0v-2z"/>
+        </svg>
+      </button>
+    </div>    
+    <div class="flex flex-grow">
+      <h1 class="w-full py-2 text-lg text-center font-semibold leading-6 tracking-tight lg:text-left">speedrun.website</h1>
+    </div>
+    <div class="flex">
+      <svg xmlns="http://www.w3.org/2000/svg" class="h-10 w-10 mx-1 lg:hidden" fill="none" viewBox="0 0 24 24" stroke="currentColor">
+        <path stroke-linecap="round" stroke-linejoin="round" stroke-width="2" d="M12 8v4l3 3m6-3a9 9 0 11-18 0 9 9 0 0118 0z" />
+      </svg>
+    </div>
+    <div class="flex flex-col w-full my-2 text-center font-semibold lg:flex-row lg:w-auto lg:my-0" v-if="mobileView || lg">
```

Comment:
You should avoid using v-if here as it disrupts the node tree on load. Of course this only matters if the component is rendered on the server, which we should do. You should instead hide the element with v-show or class binding with display classes.

Reply:
Changed ```v-if``` to ```v-show```. Due to the method of toggling elements, the nav items that are displayed with ```v-show```are currently hidden for a short period of time during site loading until the ```beforeMounted()``` hook is called. This only affects window sizes ```>= lg```. Probably something that can be optimized in the future.